packages/untp-test-suite/src/core/utils/common.ts

```diff
@@ -14,22 +14,47 @@ export const readJsonFile = async <T>(filePath: string): Promise<T> => {
   return JSON.parse(fileContent);
 };
 
+/**
+ * Map the required error object
+ * @param {string} missingField - The name of the field that is required.
+ * @returns {ErrorObject} The required error object.
+ */
+const createMissingFieldError = (credentialConfigsPath: string, missingField?: string): ICredentialConfigError => {
+  const error = {
+    message: missingField ? `should have required property '${missingField}'` : null,
+    keyword: missingField ? 'required' : null,
+    configPath: credentialConfigsPath,
```

Comment:
I think the `configPath` field here could be changed to `dataPath` to match with the `error.hbs` template. That field indicates the file path being tested.
```hbs
{
  "testingCredential": "{{type}} {{version}}",
  "path": "{{dataPath}}",  {{!-- {{This line}} --}}
  "result": "FAIL",
  {{!-- {{Others....}} --}}
}
```